AirCasting/Dashboard/DashboardView.swift

```diff
@@ -74,6 +74,12 @@ struct DashboardView: View {
             try! coreDataHook.setup(selectedSection: self.selectedSection.selectedSection)
         }
         .onAppear {
```

Comment:
Seems like it's still working when we delete this onAppear, because you've put the same in mainTapBar view